gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -53,12 +54,15 @@ def __init__(self,
         metadata values (:class:`~vcf.parser._Format`).
       contigs (dict): A dictionary mapping contig keys (str) to vcf contig
         metadata values (:class:`~vcf.parser._Contig`).
+      conflicts (dict): A dictionary mapping keys (str) to a set of conflicts
+        (set[str])
```

Comment:
The other one was to create a new class which contain a `header` and one conflicts dict. I did not go with that since we have to rewrite most of the code in `merge_headers`. Now I am thinking maybe it is better to extend the header, and then have an `conflicts` dict. WDYT?